torch/_functorch/_aot_autograd/dispatch_and_compile_graph.py

```diff
@@ -48,6 +49,13 @@ def _create_graph(f, args, *, aot_config: AOTConfig) -> torch.fx.GraphModule:
     return fx_g
 
 
+def _root_module_for_non_strict_export(flat_fn):
+    if hasattr(flat_fn, "_orig_mod") and hasattr(flat_fn._orig_mod, "_export_root"):
```

Comment:
I am a little worried we run the risk of dropping this when someone adds another layer of wrapping/tracing to AOTAutograd 🤔  (cc @tugsbayasgalan )

I guess the alternative to this though is the previous "stash info about buffer swapping on the FX graph, so we can plumb it back up to the logic at the aot_export_module top-level that generates the GraphSignature". I don't have much of a preference between that or this

Reply:
So ideally we would keep a reference to the original module somewhere, and no matter how much wrapping we do it should still be there as a stable API. I just don't know where it is or whether it even exists.

Reply:
Added comment to make this dependency explicit.